libpod/image.go

```diff
@@ -50,29 +55,116 @@ type CopyOptions struct {
 // return will include the image, a false return will exclude it.
 type ImageFilter func(*storage.Image) bool
 
+type imagePullStruct struct {
```

Comment:
Can this have a more descriptive name? "imageToPull" or something? Or a comment that makes it more clear what it's for

Reply:
fixed